sev-types/src/command/code.rs

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! The `code` module contains convenient symbolic descriptions
+//! of the possible commands that may be made to the SEV platform.
+//!
+//! Chapter 4.3, "Command Identifiers"
+
+use enumerate::enumerate;
+
+enumerate! {
+    /// Symbolic representation of the command codes exported by the
+    /// SEV API. These enums map directly to an integral value that the
+    /// SEV API calls an "ID."
+    #[derive(Copy, Clone)]
+    pub enum Code: u32 {
+
+    /// Initialize the platform.
+    Init = 0x001,
```

Comment:
No empty line before the first variant. Also, indentation is wrong for all variants.

Reply:
I agree, but `cargo fmt` does not.

Reply:
Hrm... `cargo fmt` doesn't yell at me for this in `intel-types::Exception`...